docs/src/pages/IconPage/IconPage.js

```diff
@@ -36,7 +36,9 @@ const Page = () => (
       <IntroBlurb>
         A suite of icons for representing concepts, features and actions.
         Looking for the icon set? Check out the{' '}
-        <BpkLink href="/components/icon?platform=design">design tab</BpkLink>{' '}
+        <BpkRouterLink to="/components/icon?platform=design">
+          design tab
+        </BpkRouterLink>{' '}
```

Comment:
Unrelated change - this just saves the entire page reloading when the link it used